src/test/java/com/gooddata/warehouse/WarehouseServiceIT.java

```diff
@@ -30,11 +31,13 @@
 public class WarehouseServiceIT extends AbstractGoodDataIT {
 
     private static final String TITLE = "Test";
+    private static final String SCHEMA_NAME = "default";
```

Comment:
what is the default schema? is it something always present? maybe user would use some handy get method in the service?